src/components/NavBar.jsx

```diff
@@ -0,0 +1,25 @@
+const links = [
+  { text: 'Rockets' },
+  { text: 'Mission' },
+  { text: 'My Profile' },
+];
+
+const NavBar = () => (
+  <header className="navbar-container">
+    <nav className="nav-bar">
+      <div className="logo">
+        <h1>Space Travelers&apos; Hub</h1>
+        <img src="\resource\planet.png" alt="logo" />
+      </div>
+      <ul className="nav-link">
+        {links.map((link) => (
+          <li key={link.text}>
+            {link.text}
+          </li>
+        ))}
+      </ul>
+    </nav>
+  </header>
```

Comment:
For project requirement you should use NavLink instead of link .

Reply:
Hello @YoutLone,

App.js is a parent component, so it would be obvious if we place it inside `/components` directory.
For navigation links, a separate component `</ NavLink >` has been created.